### PR TITLE
MGMT-16060: Add e2e test to aws-sno-fips-recert

### PR DESCRIPTION
**Pull request by wangke19**
Changes of this PR:
1. Change old test profile name `aws-sno-fips-recert` to  `e2e-aws-ipi-sno-fips-recert`
2. Remove the chain cucushift-installer-rehearse-aws-ipi-sno-recert-workflow
3. Add `ref: single-node-recert` step before executing the openshift-e2e-test-qe
4. Add `openshift-e2e-test-qe` test with TEST_FILTERS to the test profile.

### CODE REVIEW

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-rh-ecosystem-edge-recert-main-4.15-aws-sno-fips-recert

**Comment by xingxingxia** [comment]
/retitle [WIP] MGMT-16060: Add e2e test to aws-sno-fips-recert

**Comment by wangke19** [comment]
Many cases ran into failures, need to run manually in local.

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-rh-ecosystem-edge-recert-main-4.14-e2e-aws-ipi-sno-fips-recert

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-rh-ecosystem-edge-recert-main-4.14-e2e-aws-ipi-sno-fips-recert

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-rh-ecosystem-edge-recert-main-4.14-e2e-aws-ipi-sno-fips-recert

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-rh-ecosystem-edge-recert-main-4.14-e2e-aws-ipi-sno-fips-recert

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-rh-ecosystem-edge-recert-main-4.14-e2e-aws-ipi-sno-fips-recert

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-rh-ecosystem-edge-recert-main-4.14-e2e-aws-ipi-sno-fips-recert

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-rh-ecosystem-edge-recert-main-4.14-e2e-aws-ipi-sno-fips-recert

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-rh-ecosystem-edge-recert-main-4.14-e2e-aws-ipi-sno-fips-recert

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-rh-ecosystem-edge-recert-main-4.14-e2e-aws-ipi-sno-fips-recert

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-rh-ecosystem-edge-recert-main-4.15-e2e-aws-ipi-sno-fips-recert

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-rh-ecosystem-edge-recert-main-4.16-e2e-aws-ipi-sno-fips-recert

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-rh-ecosystem-edge-recert-main-4.15-e2e-aws-ipi-sno-fips-recert

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-rh-ecosystem-edge-recert-main-4.14-e2e-aws-ipi-sno-fips-recert

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-rh-ecosystem-edge-recert-main-4.14-e2e-aws-ipi-sno-fips-recert

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-rh-ecosystem-edge-recert-main-4.14-e2e-aws-ipi-sno-fips-recert

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-rh-ecosystem-edge-recert-main-4.14-e2e-aws-ipi-sno-fips-recert

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-rh-ecosystem-edge-recert-main-4.14-e2e-aws-ipi-sno-fips-recert

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-rh-ecosystem-edge-recert-main-4.14-e2e-aws-ipi-sno-fips-recert

**Comment by wangke19** [comment]
Checked the test results of [ci/rehearse/periodic-ci-rh-ecosystem-edge-recert-main-4.14-e2e-aws-ipi-sno-fips-recert](https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/47821/rehearse-47821-periodic-ci-rh-ecosystem-edge-recert-main-4.14-e2e-aws-ipi-sno-fips-recert/1752252864048140288) with high pass ratio 98.93 %, 
`error: 6 fail, 553 pass, 214 skip (2h24m38s)`
and file [jira issues](https://issues.redhat.com/browse/MGMT-16060?focusedId=23986348&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-23986348) to track down the failed cases. Vverall, we got the expected results.

**Comment by wangke19** [comment]
/assign @mresvanis @jianlinliu 
Please help to review the PR. thanks.

File: ci-operator/config/rh-ecosystem-edge/recert/rh-ecosystem-edge-recert-main.yaml [hunk 1]
```diff
@@ -94,17 +94,18 @@ tests:
     - chain: openshift-e2e-test-qe
     workflow: openshift-e2e-aws-single-node-recert
   timeout: 8h0m0s
-- as: aws-sno-fips-recert
+- as: e2e-aws-ipi-sno-fips-recert
```
Comment (jianlinliu) [inline ci-operator/config/rh-ecosystem-edge/recert/rh-ecosystem-edge-recert-main.yaml:97]: cc @liangxia to review.

Comment (liangxia) [reply ci-operator/config/rh-ecosystem-edge/recert/rh-ecosystem-edge-recert-main.yaml:97]: What's reason to rename this test name with prefix `e2e` ?

Comment (wangke19) [reply ci-operator/config/rh-ecosystem-edge/recert/rh-ecosystem-edge-recert-main.yaml:97]: The added execution test is `openshift-e2e-test-qe`,  so named with prefix e2e.

Comment (liangxia) [reply ci-operator/config/rh-ecosystem-edge/recert/rh-ecosystem-edge-recert-main.yaml:97]: Just found this is another team's repo/folder. I don't have objection to this. No change required.

**Comment by xingxingxia** [comment]
@wangke19 @liangxia LGTM
/lgtm

@wangke19 one question, for next PR to add Baremetal SNO recert and e2e CI, is it possible to add them under QE's folder https://github.com/openshift/release/tree/master/ci-operator/config/openshift/openshift-tests-private ?

**Comment by wangke19** [comment]
> @wangke19 @liangxia LGTM /lgtm
> 
> @wangke19 one question, for next PR to add Baremetal SNO recert and e2e CI, is it possible to add them under QE's folder https://github.com/openshift/release/tree/master/ci-operator/config/openshift/openshift-tests-private ?

Will give a try in next PR to do that.

**Pull request merged**